src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss

```diff
@@ -48,6 +48,10 @@ a {
     padding-left: .2em;
 }
 
+.ui-media-upload-button {
```

Comment:
Sounds good. But for here it only needs a `margin-bottom` instead of `margin-top` if i apply it to this class for #15909, it would change the behavior also for here:

![Bildschirmaufnahme 2024-03-01 um 13 57 47](https://github.com/Sylius/Sylius/assets/39345336/9195717f-34cf-40d2-b1f6-8daace9433ca)

I'm not sure if we really want it too look like that afterwards 🤔

Reply:
Also:

The media tab in the Taxon module looks a bit different than in the product media tab. If i would apply my `ui-element-spacer` class there, it would look broke first:

![SCR-20240301-mqhj](https://github.com/Sylius/Sylius/assets/39345336/daaf4e8e-5e20-4234-af8f-6a4f3f6a53f8)

So to make it look consistent, i would suggest to make it look like in the product media tab:

![SCR-20240301-mqsl](https://github.com/Sylius/Sylius/assets/39345336/3acba695-634d-465a-87b3-fa74e5245aca)

Wdyt?

Reply:
We discussed this within our team and concluded that your present solution is acceptable 🕺